Editors/SpriteEditor.ui

```diff
@@ -127,6 +202,9 @@
             <property name="text">
              <string>&amp;Y</string>
             </property>
+            <property name="alignment">
+             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>

```

Comment:
I liked these better left-aligned

Reply:
I have to agree with you actually, I'll think about it before I merge. As a matter of technicality though, they were right aligned in GM.
![GM BBox Labels](https://user-images.githubusercontent.com/3212801/90996597-3e80cf80-e58d-11ea-95a9-e6fdff1ae638.png)

Reply:
Alright I sent 71843382a789008d6c98d9feb5250389508bf849 to put it back for now.